test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishToClickOnce.cs

```diff
@@ -65,13 +65,20 @@ public void PublishClickOnceWithPublishProfile(bool? publishSingleFile)
     {(publishSingleFile.HasValue ? $"<PublishSingleFile>{publishSingleFile}</PublishSingleFile>" : "")}
     {(publishSingleFile == true ? $"<RuntimeIdentifier>{rid}</RuntimeIdentifier>" : "")}
   </PropertyGroup>
+  <ItemGroup>
+    <BootstrapperPackage Include=""Microsoft.SqlServer.SqlLocalDB.11.1"">
+      <Install>true</Install>
+      <ProductName>SQL Server 2012 Express LocalDB</ProductName>
```

Comment:
I would recommend using a newer package that is guaranteed to be around for a while like .NET 4.8.1 if you can.  This old SQL Express package maybe deleted soon.

Reply:
fixed.